src/app/components/header/header.component.spec.ts

```diff
@@ -154,4 +158,22 @@ describe('HeaderComponent', () => {
             fixture.debugElement.query(By.css('.header__alert-box'))
         ).toBeFalsy();
     });
+
+    it('should not show a notifications badge by default', () => {
+        expect(
+            fixture.debugElement.query(By.css('.mat-badge-hidden'))
+        ).toBeTruthy();
+    });
+
+    it('should display the number of notifications in a badge', () => {
+        notificationService.addInfoNotification({
+            title: 'Testing',
+            description: 'Currently testing the application.',
+            icon: '',
+        }),
+            fixture.detectChanges();
```

Comment:
Some weird formatting here, are tests not linted?

Reply:
They are. Weird that this was not found